stdlib/public/Concurrency/SuspendingClock.swift

```diff
@@ -0,0 +1,144 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Swift
+
+@available(SwiftStdlib 9999, *)
+public struct SuspendingClock {
+  public struct Instant: Codable, Sendable {
+    internal var _value: Swift.Duration
+
+    internal init(_value: Swift.Duration) {
+      self._value = _value
+    }
+  }
+
+  public init() { }
+}
+
+@available(SwiftStdlib 9999, *)
+extension Clock where Self == SuspendingClock {
+  @available(SwiftStdlib 9999, *)
+  public static var suspending: SuspendingClock { return SuspendingClock() }
+}
+
+@available(SwiftStdlib 9999, *)
+extension SuspendingClock: Clock {
+  @available(SwiftStdlib 9999, *)
+  public var now: SuspendingClock.Instant {
+    SuspendingClock.now
+  }
+
+  @available(SwiftStdlib 9999, *)
+  public static var now: SuspendingClock.Instant {
+    var seconds = Int64(0)
+    var nanoseconds = Int64(0)
+    _getClockRes(
```

Comment:
Same suspicion about `getTime` vs `_getClockRes` here

Reply:
yep - also got that one.